src/components/Form/ToggleButton.tsx

```diff
@@ -55,7 +55,7 @@ export class ToggleButton extends Component<ToggleProps> {
 
     if (!!toggleInputProps || !!toggleInputComponent) {
       logger.error(
-        `ToggleButton does not utilize 'toggleInputProps' and 'toggleInputComponent' props.`,
+        'ToggleButton does not utilize toggleInputProps and toggleInputComponent props.',
```

Comment:
Escaping characters should be possible here? \'

Reply:
I changed this when looking through the warning given by eslint and thought that there might be no reason to use single quotes around the props to give anything more to the reader of warning message.

This current eslint rule allows also the previous version. 
https://github.com/vrk-kpa/suomifi-ui-components/blob/dca31e6cc60c34523004d1c599ee5102cfba084d/.eslintrc#L100

Maybe we can return this highlighted code to its previous value if it is giving more to the reader?

Reply:
Reverted to the previous value in latest commit.